tests/virtctl/usbredir.go

```diff
@@ -187,79 +171,110 @@ var _ = Describe("[crit:medium][vendor:cnv-qe@redhat.com][level:component][sig-c
 })
 
 func runConnectGoroutine(
-	virtClient kubecli.KubevirtClient,
 	name string,
 	namespace string,
 	ctx context.Context,
 	errch chan error,
 ) {
-	defer GinkgoRecover()
-
-	usbredirStream, err := virtClient.VirtualMachineInstance(namespace).USBRedir(name)
-	if err != nil {
-		errch <- err
-		return
-	}
-	usbredirConnect(usbredirStream, ctx)
-}
-
-func usbredirConnect(
-	stream kvcorev1.StreamInterface,
-	ctx context.Context,
-) {
-
-	usbredirClient, err := usbredir.NewUSBRedirClient(ctx, "localhost:0", stream)
-	Expect(err).ToNot(HaveOccurred())
-
-	usbredirClient.ClientConnect = func(inCtx context.Context, device, address string) error {
+	cmd := newVirtctlCommand("usbredir",
+		"--namespace", namespace,
+		"--no-launch",
+		name,
+	)
+	// To find ip/port to connect
+	rOut, wOut := io.Pipe()
+	cmd.SetOut(wOut)
```

Comment:
Does the cmd close the pipes on exit? Otherwise it might be better to close them in the scope of this function instead of closing them with `defer r.Close()` in the go routines, so we are safe we do not leak the routines?

Reply:
Sure

Reply:
Missed this one, running locally now and I'll push shortly..